db/schema_tables.cc

```diff
@@ -830,29 +831,21 @@ future<table_schema_version> calculate_schema_digest(distributed<service::storag
     return calculate_schema_digest(proxy, features, std::not_fn(&is_system_keyspace));
 }
 
-future<std::vector<canonical_mutation>> convert_schema_to_mutations(distributed<service::storage_proxy>& proxy, schema_features features)
+future<> convert_schema_to_mutations(distributed<service::storage_proxy>& proxy, schema_features features, std::function<void(mutation)> process_mutation)
 {
-    auto map = [&proxy, features] (sstring table) -> future<std::vector<canonical_mutation>> {
-        auto& db = proxy.local().get_db();
+    auto& db = proxy.local().get_db();
+    for (const auto& table : all_table_names(features)) {
```

Comment:
There is loss of parallelism here, map_reduce() maps in parallel.

Reply:
What parallelism? unfreezing_gently is cpu-bound, preemption would be due to maybe_yield, so I don't see what that buys us.

Reply:
Parallelism of db::system_keyspace::query_mutations(), which may go to disk.

Reply:
ok